src/vos/vos_pool.c

```diff
@@ -415,6 +447,7 @@ vos_wal_commit(struct umem_store *store, struct umem_wal_tx *wal_tx, void *data_
 	} else if (vwm != NULL) {
 		d_tm_set_gauge(vwm->vwm_wal_sz, ws.ws_size);
 		d_tm_set_gauge(vwm->vwm_wal_qd, ws.ws_qd);
+		d_tm_set_gauge(vwm->vwm_wal_lat[d_tm_io_lat_bucket(ws.ws_size)], time);
```

Comment:
The 'ws_size' is WAL I/O size, it's different from the "I/O size". As I mentioned above, adding a "wal latency" should be sufficient.

Reply:
- [ ] Use single WAL latency

Reply:
Fixed with commit https://github.com/daos-stack/daos/commit/9ed8e926e187507488268de28f14bca8d3a5af36